storage/response_cache.go

```diff
@@ -246,29 +200,6 @@ func (rc *ResponseCache) shouldCacheResponse(reply proto.Response) bool {
 	return true
 }
 
-// addInflightLocked adds the supplied ClientCmdID to the inflight
-// map. Any subsequent invocations of GetResponse for the same client
-// command will block on the inflight cond var until either the
-// response cache is cleared or this command is removed via
-// PutResponse().
-func (rc *ResponseCache) addInflightLocked(cmdID proto.ClientCmdID) {
```

Comment:
the idea here is that there's no point in having the inflight map any more since the leader lease mutex holds up concurrent requests while the lease is acquired, correct?